### ISSUE TITLE
Log can fill database

### ISSUE THREAD

**Issue by kichik** [yunojuno/django-request-token#53]
It seems like there is no way to disable the logging option that writes an entry to the database for every use. We found that out because on Heroku you get limited after writing 10k rows and we got there really fast on our test website. This log can add up quickly on a big website and it doesn't seem like there is a way to limit it or give it TTL.

https://github.com/yunojuno/django-request-token/blob/master/request_token/models.py#L310-L327

https://github.com/yunojuno/django-request-token/blob/7580ede55a3bd2826980651bbf3843078e0574a7/request_token/decorators.py#L72-L81

**Comment by hugorodgerbrown** [comment]
Hi @kichik - thanks for raising this. I'm not a great fan of a global switch for logging use, as the audit trail is an integral part of the library (certainly for our use case), and it's proven invaluable from a customer support perspective - I think people _should_ have it on. That said, controlling the logging is clearly an issue if you are constrained. A management command to clear out old / excessive log entries would be my preferred option.

**Comment by hugorodgerbrown** [comment]
PR #54 adds a management command to restrict growth of the log.

**Comment by kichik** [comment]
We ended up with this workaround:

```
class MyAppConfig(AppConfig):
    name = 'myapp'

    def ready(self):
        def fake_log(*args, **kwargs):
            # don't log and fill the database
            # https://github.com/yunojuno/django-request-token/issues/53
            pass

        from request_token.models import RequestToken
        RequestToken.log = fake_log
```

**Comment by hugorodgerbrown** [comment]
The next release will have a `DISABLE_LOGS` setting. No timeframe on the release as yet.

**Comment by hugorodgerbrown** [comment]
This is now available as v0.16.
